.gitignore

```diff
@@ -67,8 +67,8 @@ conf/locale/messages.mo
 .noseids
 nosetests.xml
 .cache/
-.coverage
-.coverage.*
+# .coverage
+# .coverage.*
```

Comment:
I don't understand why you would want to remove these files from .gitignore?

Reply:
Thanks for noticing this, actually I've not finalised this PR yet, it's work is in progress. I will let you know once I finish work on this PR.
@nedbat